.github/workflows/sync-internal-and-public.yml

```diff
@@ -41,7 +41,9 @@ jobs:
           cp -a internal/clients temp/clients
           cp -a internal/docs temp/docs
           cp -a internal/scripts temp/scripts
-          cp -a internal/cmd/transform-swagger temp/cmd/transform-swagger
+          mkdir temp/cmd
```

Comment:
would it make sense to combine these two mkdir calls into 1 with `mkdir -p tmp/cmd/transform-swagger`

Reply:
Ah thanks for the suggestion! My bash skills are a little rusty 😅